content/platform/zerto/home.md

```diff
@@ -1,34 +1,46 @@
 
-### Zerto for Kubernetes
+### Simplify Automation with Zerto’s REST Swagger APIs
 
-Zerto for Kubernetes (Z4K) enables continuous backup, disaster recovery, and long-term retention for stateful and stateless Kubernetes applications running on-premises or in the cloud.
+Zerto’s REST Swagger APIs offer a powerful and user-friendly way to automate and integrate Zerto into your existing IT workflows. These APIs provide a comprehensive set of endpoints that allow you to programmatically manage, monitor, and control your Zerto environment with ease. Whether you are looking to automate routine tasks, integrate with third-party tools, or develop custom applications, Zerto’s REST APIs make it straightforward and efficient.
 
-Learn more by watching Zerto for Kubernetes Overview 
+With Swagger, developers can interactively explore and test the APIs, making it easier to understand their capabilities and integrate them into your systems. The Swagger interface provides clear documentation and real-time testing capabilities, ensuring that you can quickly get up to speed and start leveraging Zerto’s powerful features.
 
-[![Zerto for Kubernetes Overview](https://img.youtube.com/vi/EIQcOIcbBwU/hqdefault.jpg)](https://www.youtube.com/watch?v=EIQcOIcbBwU)
+Unlock the full potential of your disaster recovery strategy with Zerto’s REST Swagger APIs today!
+
+For more information and to access the Zerto REST Swagger APIs, visit the [Zerto API Documentation](https://www.zerto.com/myzerto/apis/).
+
+
+
+### Unlock Powerful Insights with Zerto Resiliency Observation Console
+
+Meet the Zerto Resiliency Observation Console (zROC) – a dynamic, Docker-compose based software stack designed to transform your Zerto API data into rich, visual insights using Prometheus and Grafana. This innovative tool empowers you to monitor and analyze your data effortlessly.
+
+What sets zROC apart is its custom Prometheus exporter code, crafted to seamlessly integrate with standard Prometheus and Grafana containers. The comprehensive configuration files included in the repository ensure that you can start deriving value immediately, straight out of the box.
+
+While zROC comes with pre-built dashboards to get you started, we encourage you to unleash your creativity by building custom dashboards or tweaking existing ones to perfectly match your unique needs. Dive into the world of intuitive data visualization and take your monitoring capabilities to the next level.
+
+Explore the Zerto Resiliency Observation Console today on our [GitHub page](https://github.com/ZertoPublic/zroc).
 
-or going to our [solution page](https://www.zerto.com/solutions/workloads-and-applications/zerto-for-kubernetes/).
 
-Follow along with on the [Zerto for Kubernetes GitHub](https://github.com/ZertoPublic/Z4K/blob/main/docs/DeployingZertoForK8S.md) page to enjoy disaster recovery and data mobility of your K8s applications.
 
 ### Zerto In Cloud for AWS
```

Comment:
I see you writing it as Zerto In-Cloud in other areas. It's fine to do that in the subheader here, too. Just make sure you're consistent everywhere.


Hmm... something appears to be missing here. I see the header, but I'm not seeing these lines that I see in the preview:

![image](https://github.com/user-attachments/assets/e6649c3b-a70a-4c50-9136-213efd2db975)


The "Instances" was something I wanted to comment on. I believe the "I" should be a lower-case "i" here, unless this is a proper noun/name of the product.